npc/quests/quests_ein.txt

```diff
@@ -474,20 +474,481 @@ einbech,97,167,5	script	Cavitar	4_M_EINOLD,{
+		mes "skills, you can't let yourself be discouraged! Practice makes";
+		mes "perfect, you know? But never";
+		mes "use your lack of skills as an";
+		mes "excuse if you happen to fail...";
+		next;
+		mes "[Uwe]";
+		mes "Effort is also an essential";
+		mes "in forging and cooking! Now,";
+		mes "on the other hand, if you put";
+		mes "in all the effort but didn't learn any of the skills, you'll still get nowhere fast, right? Right!";
+		next;
+		mes "[Uwe]";
+		mes "The key is to always";
+		mes "give 100% effort and work";
+		mes "on improving your skills.";
+		mes "Before you know it, you'll";
+		mes "be a respected master!";
+		next;
+		mes "[Uwe]";
+		mes "Well, that was";
+		mes "a fun talk. Come";
+		mes "back and visit, 'kay?";
+		mes "I'll miss you until";
+		mes "the next time~";
+		F_RandCloseMsg();
+	}
+	
+	function F_Tools {
+		mes "[Uwe]";
+		mes "Cooking is a little";
+		mes "easier if you have";
+		mes "nicer tools to use,";
+		mes "but that's it. Tools by";
+		mes "themselves can't make";
+		mes "just anybody a master.";
+		next;
+		mes "[Uwe]";
+		mes "In the end, the best";
+		mes "tools are the ones you're";
+		mes "most comfortable with using.";
+		mes "In fact, I still use the old knife I used back when I was just";
+		mes "a little novice chef~";
+		next;
+		mes "[Uwe]";
+		mes "Oh, while we're on";
+		mes "the subject of tools,";
+		mes "I just received some";
+		mes "new hammers today.";
+		mes "Would you like a couple?";
+		next;
+		if (checkweight(Spawn, 50) == 0) {
+			mes "[Uwe]";
+			mes "Mmmm...?";
+			mes "You're holding too many";
+			mes "things. I can't give you any";
+			mes "hammers if you don't have";
+			mes "room. Hurry, put your extra";
+			mes "stuff in your Kafra Storage~";
+			close;
+		}
+		ein_cook += 10;
+		getitem Golden_Hammer, 1;
+		getitem Iron_Hammer, 1;
+		next;
+		mes "[Uwe]";
+		mes "Well, I hope";
+		mes "you like them~";
+		F_RandCloseMsg();
+	}
+	
+	function F_ToolsWithoutReward {
+		mes "[Uwe]";
+		mes "Cooking is a little";
+		mes "easier if you have";
+		mes "nicer tools to use,";
+		mes "but that's it. Tools by";
+		mes "themselves can't make";
+		mes "just anybody a master.";
+		next;
+		mes "[Uwe]";
+		mes "In the end, the best";
+		mes "tools are the ones you're";
+		mes "most comfortable with using.";
+		mes "In fact, I still use the old knife I used back when I was just";
+		mes "a little novice chef~";
+		next;
+		mes "[Uwe]";
+		mes "Well, that was";
+		mes "a fun talk. Come";
+		mes "back and visit, 'kay?";
+		mes "I'll miss you until";
+		mes "the next time~";
+		F_RandCloseMsg();
+	}
+	
+	function F_OverWeight {
+		mes "[Uwe]";
+		mes "Oh, I was going to give";
+		mes "you a little something,";
+		mes "but you've carrying too";
+		mes "many things. Why don't";
+		mes "you put some of it away";
+		mes "in your Kafra Storage?";
+		close;
+	}
+	
+	function F_LargeJellopyForCoal {
+		if (countitem(Large_Jellopy) > 5) {
+			if (checkweight(Spawn, 199) == 0) {
```

Comment:
Curious why you actually did if (){} here when in the previous section above you didn't.